string contur_id
int8 x
int8 y
---
string contur_id
string error_code
string error_description
geometry_msgs/Point[] points